Makefile

```diff
@@ -28,6 +28,33 @@ all: $(TARGET)
 test: LDFLAGS += $(TEST_COV)
 test: integration_test
 
+# Build Lightning binary
+build: Dockerfile
+	# Create image for compiling Lightning under Ubuntu 14.04/Trusty
+	docker build -t lightning.build .
+	# Output tarballed Lightning binary
+	docker run --rm lightning.build > lightning.tar
+
+# Deploy Lightning binary
+deploy: Dockerfile.run lightning.tar
+	# Copy over binary + config + test files
+	rm -rf deploy
+	mkdir deploy
+	tar -xf lightning.tar
+	chmod 0755 lightning
+	cp lightning deploy
+	cp Dockerfile.run deploy
+	cp simple_config deploy
+	cp -r test deploy
+	# Create image for running Lightning under BusyBox and run it!
+	# Note that make executes each command in a new subshell,
+	# and we need this to all happen in the same folder
```

Comment:
Can maybe make it a little more explicit in the comment that 'this' is the semicolon, even though the stackoverflow post is linked.